docs/admin/misc_configuration_tips.md

```diff
@@ -19,6 +19,22 @@ If running Stellio in Kubernetes, it can be configured in the deployments:
   value: 512KB
 ```
 
+## Increase the default and maximum limit for pagination
+
+Stellio has a default pagination limit of 30 and a maximum of 100. 
+
+These values can be changed by configuring the `application.pagination.limit-default` and `application.pagination.limit-max` properties in `shared.properties` file.
+
+If running Stellio from `docker-compose`, it can be configured in the environment section of the root project : 
```

Comment:
```suggestion
If running Stellio from `docker-compose`, it can be configured in the environment section of the target service (`search-service` or `subscription-service`) : 
```